src/actions/vstudio/vs2010_vcxproj_user.lua

```diff
@@ -87,7 +87,12 @@
 
 	function m.localDebuggerCommand(cfg)
 		if cfg.debugcommand then
-			local dir = p.project.getrelative(cfg.project, cfg.debugcommand)
+			if not path.isabsolute(cfg.debugcommand) then
+				print("Error: debugcommand must be an absolute path")

```

Comment:
Premake makes all paths absolute automatically, so this should never happen. It should be enough to remove the call to getrelative().